src/components/common/Search/Search.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
Duplicated Search componente here again, you can probably just delete it, it's in `src/components/ui` in master.